File: challenges/2-idade-em-dias/C/emptybrain/idade_em_dias.c
#include "stdio.h"

int main() {
  printf("Digite os dias: ");
  int n_days;
  scanf("%d", &n_days);
  int anos = (n_days) / 365;
  int meses = (n_days % 365) / 30;
  int dias = (n_days % 365) % 30;
  printf("\n ano(s): %d \n mes(es): %d \n dia(s): %d \n", anos, meses, dias);
}

File: challenges/2-idade-em-dias/java/mateussnts/IdadeEmDias.java
import java.util.Scanner;

public class IdadeEmDias {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.printf("Digite aqui sua Idade em Dias: ");
        int idade = sc.nextInt();

        int anos = idade / 365;
        int meses = (idade % 365) / 30;
        int dias = (idade % 365) % 30;

        System.out.println("\nResultados: ");
        System.out.println(anos + " ano(s)");
        System.out.println(meses + " mes(es)");
        System.out.println(dias + " dia(s)");

        sc.close();
    }
}
